apps/functions-subscription/src/domain/trial.ts

```diff
@@ -0,0 +1,42 @@
+import * as t from 'io-ts';
+import * as TE from 'fp-ts/lib/TaskEither';
+// TODO: Move this codec into this file later
```

Comment:
**note**: I'm going to move this import once the PR #96 has been merged.
That PR made some changes on the same files I'm going to update.